.github/workflows/continuous-integration-workflow.yml

```diff
@@ -12,78 +12,59 @@ on:
       - develop
 
 jobs:
```

Comment:
You could also add a rule enforcer reusable workflow job like we try to do in other apps.

Reply:
Done :+1: